### PR TITLE
Merge custom node examples from develop.

### PR DESCRIPTION
**Pull request by rasapala**

### CODE REVIEW

File: src/custom_nodes/east_ocr/east_ocr.cpp [hunk 1]
```diff
@@ -291,6 +291,9 @@ int execute(const struct CustomNodeTensor* inputs, int inputsCount, struct Custo
             NODE_ASSERT(x2 > x1, "detected box width must be greater than 0");
             NODE_ASSERT(y2 > y1, "detected box height must be greater than 0");
 
+            NODE_ASSERT(x2 > x1, "detected box width must be greater than 0");
```
Comment (dkalinowski) [inline src/custom_nodes/east_ocr/east_ocr.cpp:294]: Lines 291-292 and 294-295 are the same.

File: src/custom_nodes/image_transformation/image_transformation.cpp [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+#include <map>
+#include <string>
+
+#include "../../custom_node_interface.h"
+#include "opencv2/opencv.hpp"
+#include "utils.hpp"
+
+static constexpr const char* TENSOR_NAME = "image";
+
+template <class TimeT = std::chrono::milliseconds,
```
Comment (dkalinowski) [inline src/custom_nodes/image_transformation/image_transformation.cpp:27]: This code needs to be scanned with IP scan.

**Comment by dkalinowski** [comment]
This fix also needs to be merged in as it fixes missing exception handling:
https://github.com/openvinotoolkit/model_server/pull/634

**Comment by dkalinowski** [comment]
This commit is missing:
https://github.com/openvinotoolkit/model_server/commit/26b9efe7fd5fe818ebf261e50422d006e20b55d6

**Comment by dkalinowski** [comment]
Before merge I'd manually go through all pipeline examples and see if it compiles and runs. I'd check if pythonic example clients work aswell. Same as usually done by validation team before release.

File: src/custom_nodes/image_transformation/image_transformation.cpp (hunk 2 above)
Comment (dkalinowski) [reply src/custom_nodes/image_transformation/image_transformation.cpp]: Clock timer removed.

**Review by dtrawins — APPROVED** [review]

**Review by atobiszei — APPROVED** [review]

**Pull request merged**